integration-tests/support/pages/BuildApplicationWithGitSourcePage.ts

```diff
@@ -0,0 +1,43 @@
+import { createApplicationPagePO } from '../pageObjects/createApplication-po';
+import { buildApplicationWithGitSourcePagePO } from '../pageObjects/createApplication-po';
+import { AbstractWizardPage } from './AbstractWizardPage';
+export class BuildApplicationWithGitSourcePage extends AbstractWizardPage {
```

Comment:
What about merging this class with [https://github.com/openshift/hac-dev/blob/main/integration-tests/support/pages/AddComponentPage.ts](https://github.com/openshift/hac-dev/blob/main/integration-tests/support/pages/AddComponentPage.ts ) as it already contains some logic from the same page?

Reply:
Sure, I've deleted this file and moved the methods to the AddComponentPage class. Can you please take a look again and see if it looks good?

Reply:
Nice, thanks :+1: